### PR TITLE
[LLM] New docs

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Comment by tgenaitay** [comment]
While we are at it, perhaps we should initiate a new product category and new product in the changelog menu, aligned with the public doc menu?

**Comment by bene2k1** [comment]
@tgenaitay Yes. Good idea. I will add it to the changelog and also add a category for tutorials

**Comment by bene2k1** [comment]
Changelog: 
image
Tutorials:
image

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+meta:
+  title:  Mixtral-8x7B Large Language Model
+  description: This page explains the  Mixtral-8x7B Large Language Model
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:4]:
```suggestion
  description: This page explains the Mixtral-8x7B Large Language Model
```

**Review by jcirinosclwy — APPROVED** [review]

**Comment by bene2k1** [comment]
Do not merge before "go" from @tgenaitay

**Comment by tgenaitay** [comment]
Added several changes through a new branch and pull request https://github.com/scaleway/docs-content/pull/2905
Maybe we could merge my PR and consolidate my commits to the current pull request here?

File: ai-data/llm-inference/concepts.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  title: LLM Endpoints - Concepts
+  description: This page explains all the concepts related to Inference for LLMs
+content:
+  h1: LLM Endpoints - Concepts
+  paragraph: This page explains all the concepts related to Inference for LLMs
+tags: 
+categories:
+  - ia
+---
+## Allowed IPs
```
Comment (nerda-codes) [inline ai-data/llm-inference/concepts.mdx:12]:
```suggestion
## Access Control List rules
```

File: ai-data/llm-inference/concepts.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+A deployment makes a trained language model available for real-world applications. It encompasses tasks such as integrating the model into existing systems, optimizing its performance, and ensuring scalability and reliability.
+
+## Embedding Models
+
+Embedding models are a representation-learning technique that converts textual data into numerical vectors. These vectors capture semantic information about the text and are often used as input to downstream machine-learning models or algorithms.
+
+## Endpoint
+
+An endpoint in the context of large language models refers to a network-accessible URL or interface through which clients can interact with the model for inference tasks. It exposes methods for sending input data and receiving model predictions or responses.
```
Comment (nerda-codes) [inline ai-data/llm-inference/concepts.mdx:32]:
```suggestion
In the context of large language models, an endpoint refers to a network-accessible URL or interface through which clients can interact with the model for inference tasks. It exposes methods for sending input data and receiving model predictions or responses.
```

File: ai-data/llm-inference/concepts.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Large Language Models (LLMs) are advanced artificial intelligence systems capable of understanding and generating human-like text on various topics.
+These models, such as GPT-3, are trained on vast amounts of data to learn the patterns and structures of language, enabling them to generate coherent and contextually relevant responses to queries or prompts.
+LLMs have applications in natural language processing, text generation, translation, and other tasks requiring sophisticated language understanding and production.
+
+## Prompt
+
+In the context of Large Language Models (LLMs), a prompt refers to the input provided to the model to generate a desired response.
+It typically consists of a sentence, paragraph, or series of keywords or instructions that guide the model in producing text relevant to the given context or task.
+The quality and specificity of the prompt greatly influence the generated output, as the model uses it to understand the user's intent and create responses accordingly.
```
Comment (nerda-codes) [inline ai-data/llm-inference/concepts.mdx:62]:
```suggestion
The quality and specificity of the prompt greatly influences the generated output, as the model uses it to understand the user's intent and create responses accordingly.
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: How to create an deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:3]:
```suggestion
  title: How to create a deployment
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  description: This page explains how to create an deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:4]:
```suggestion
  description: This page explains how to create a deployment
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+content:
+  h1: How to create an deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:6]:
```suggestion
  h1: How to create a deployment
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  paragraph: This page explains how to create an deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:7]:
```suggestion
  paragraph: This page explains how to create a deployment
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+tags: dedibox failover ip flexible-ip failover-ip
+dates:
+  validation: 2023-09-25
+  posted: 2022-03-24
+---
+
+1. Go to the **AI & Data** section of the [Scaleway console](https://console.scaleway.com/), and select **LLM Inference** from the side menu to access the LLM Inference dashboard.
+2. Click on **Create deployment** to launch the deployment creation wizard.
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:15]:
```suggestion
2. Click **Create deployment** to launch the deployment creation wizard.
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 10]
```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: How to delete a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:3]:
```suggestion
  title: How to delete an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+  description: This page explains how to deletee a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:4]:
```suggestion
  description: This page explains how to delete an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+content:
+  h1: How to delete a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:6]:
```suggestion
  h1: How to delete an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+  paragraph: This page explains how to deletee a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:7]:
```suggestion
  paragraph: This page explains how to delete an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+<Macro id="requirements" />
+
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+
+
+1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
+2. Choose a deployment either by clicking its name or selecting **More info** from the drop-down menu represented by the icon <Icon name="more" /> to access the deployment dashboard.
+3. Click the **Settings** tab of your deployment to display additional settings.
+4. Click **Delete deployment** to delete your deployment.
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:27]:
```suggestion
4. Click **Delete deployment**.
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 15]
```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: How to monitor a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:3]:
```suggestion
  title: How to monitor an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+  description: This page explains how to monitore a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:4]:
```suggestion
  description: This page explains how to monitor an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+content:
+  h1: How to monitor a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:6]:
```suggestion
  h1: How to monitor an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+  paragraph: This page explains how to monitore a LLM Inference deployment
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:7]:
```suggestion
  paragraph: This page explains how to monitor an LLM Inference deployment
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+tags: ai monitoring
+dates:
+  validation: 2024-03-06
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Monitoring allows you to monitor your deployment.
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:16]:
```suggestion
This documentation page shows you how to monitor your LLM Inference deployment with [Cockpit](/observability/cockpit/quickstart/).
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+  - An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+
+
+## How to monitor your LLM dashboard
+
+1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
+2. Click a deployment name or <Icon name="more" /> > **More info** to access the deployment dashboard.
+3. Click the **Monitoring** tab of your deployment. The Cockpit overview displays.
+4. Click **Open Grafana metrics dashboard** to open your Cockpit's Grafana interface.
+5. Authenticate with your Grafana credentials. The Grafana dashboard displays.
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:31]:
```suggestion
5. Authenticate with your [Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/). The Grafana dashboard displays.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+  description: This page explains all the concepts related to Inference for LLMs
+content:
+  h1: LLM Endpoints - Quickstart
+  paragraph: This page explains all the concepts related to Inference for LLMs
+tags: 
+categories:
+  - ia
+---
+
+Scaleway LLM Inference service, a scalable and secure inference engine for machine learning models.
```
Comment (nerda-codes) [inline ai-data/llm-inference/quickstart.mdx:13]:
```suggestion
The Scaleway LLM Inference service is a scalable and secure inference engine for machine learning models.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+
+* **Easy deployment**: Deploy your inference models with just a few clicks. Scaleway Inference provides a simple and intuitive interface for uploading and deploying models.
+* **Auto-scaling**: Scaleway LLM Inference automatically scales your inference instances based on the demand, ensuring that your models are always available and responsive.
+* **Security**: Scaleway provides a secure environment for running your inference models. Our platform is built on top of a secure architecture, and we use state-of-the-art security.
+
+## How to create a LLM Inference deployment
+
+
+1. Navigate to the **AI & Data** section of the [Scaleway console](https://console.scaleway.com/), and select **LLM Inference** from the side menu to access the LLM Inference dashboard.
+2. Click on **Create deployment** to launch the deployment creation wizard.
```
Comment (nerda-codes) [inline ai-data/llm-inference/quickstart.mdx:28]:
```suggestion
2. Click **Create deployment** to launch the deployment creation wizard.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 23]
```diff
@@ -0,0 +1,76 @@
...
+1. Click **LLM Inference** in the **AI & Data** section of the side menu. The LLM Inference dashboard displays.
+2. Click <Icon name="more" /> next to the deployment you want to edit. The deployment dashboard displays.
+3. Click the **Inference** tab. Code examples in various languages displays. Copy and paste them in your code.
+
+## How to delete a deployment
+
+1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
+2. Choose a deployment either by clicking its name or selecting **More info** from the drop-down menu represented by the icon <Icon name="more" /> to access the deployment dashboard.
+3. Click the **Settings** tab of your deployment to display additional settings.
+4. Click **Delete deployment** to delete your deployment.
```
Comment (nerda-codes) [inline ai-data/llm-inference/quickstart.mdx:71]:
```suggestion
4. Click **Delete deployment**.
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 24]
```diff
@@ -0,0 +1,75 @@
+---
+meta:
+  title: Llama-2-70b-chat model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:3]:
```suggestion
  title: Understanding the Llama-2-70b-chat model
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 25]
```diff
@@ -0,0 +1,75 @@
...
+  description: This page explains the Llama-2-70b model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:4]:
```suggestion
  description: This page provides information on the Llama-2-70b model
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 26]
```diff
@@ -0,0 +1,75 @@
...
+content:
+  h1: Llama-2-70b model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:6]:
```suggestion
  h1: Understanding the Llama-2-70b-chat model
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 27]
```diff
@@ -0,0 +1,75 @@
...
+  paragraph: This page explains the Llama-2-70b model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:7]:
```suggestion
  paragraph: This page provides information on the Llama-2-70b model
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 28]
```diff
@@ -0,0 +1,75 @@
...
+```bash
+llama-2-70b-chat
+```
+
+## Compatible Instances
+
+- [H100 (FP8)](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+- [H100-2 (FP16)](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+
+## Model introduction
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:34]: This should be the first content to appear on this page imho

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 29]
```diff
@@ -0,0 +1,75 @@
...
+
+<Message type="tip">
+  Ensure you have the necessary libraries provided by NVIDIA for Triton to interact with the server from your client code.
+  You can find these libraries on [GitHub](https://github.com/triton-inference-server/client/tree/main).
+  Follow the instructions provided in the `README` file to integrate these libraries into your project.
+</Message>
+
+### Sending Inference requests
+
+To perform inference tasks with your LLM model deployed on the Triton Inference Server, follow these steps:
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:56]:
```suggestion
To perform inference tasks with your LLM model deployed on the Triton Inference Server, use the following command:
```

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 30]
```diff
@@ -0,0 +1,70 @@
+---
+meta:
+  title: Mixtral-8x7B-Instruct-v0.1 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:3]:
```suggestion
  title: Understanding the Mixtral-8x7B-Instruct-v0.1 model
```

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 31]
```diff
@@ -0,0 +1,70 @@
...
+  description: This page explains the Mixtral-8x7B-Instruct-v0.1 model
+content:
+  h1:  Mixtral-8x7B-Instruct-v0.1 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:6]:
```suggestion
  h1:  Understanding the Mixtral-8x7B-Instruct-v0.1 model
```

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 32]
```diff
@@ -0,0 +1,70 @@
+---
+meta:
+  title: Mixtral-8x7B-Instruct-v0.1 model
+  description: This page explains the Mixtral-8x7B-Instruct-v0.1 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:4]:
```suggestion
  description: This page provides information on the Mixtral-8x7B-Instruct-v0.1 model
```

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 33]
```diff
@@ -0,0 +1,70 @@
...
+content:
+  h1:  Mixtral-8x7B-Instruct-v0.1 model
+  paragraph: This page explains the Mixtral-8x7B-Instruct-v0.1 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:7]:
```suggestion
  paragraph: This page provides information on the Mixtral-8x7B-Instruct-v0.1 model
```

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 34]
```diff
@@ -0,0 +1,70 @@
...
+```bash
+mixtral-8x7B-Instruct-v0.1
+```
+
+## Compatible Instances
+
+- [H100-2 (FP16)](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+
+## Model introduction
+
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:32]: Same comment as above

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 35]
```diff
@@ -0,0 +1,70 @@
...
+<Message type="tip">
+  Ensure you have the necessary libraries provided by NVIDIA for Triton to interact with the server from your client code.
+  You can find these libraries on [GitHub](https://github.com/triton-inference-server/client/tree/main).
+  Follow the instructions provided in the `README` file to integrate these libraries into your project.
+</Message>
+
+### Sending Inference requests
+
+To perform inference tasks with your LLM model deployed on the Triton Inference Server, follow these steps:
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:52]:
```suggestion
To perform inference tasks with your LLM model deployed on the Triton Inference Server, use the following command:
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 36]
```diff
@@ -0,0 +1,69 @@
+---
+meta:
+  title: WizardLM-70B-V1.0 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:3]:
```suggestion
  title: Understanding the WizardLM-70B-V1.0 model
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 37]
```diff
@@ -0,0 +1,69 @@
...
+  description: This page explains the WizardLM-70B-V1.0 model
+content:
+  h1: WizardLM-70B-V1.0 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:6]:
```suggestion
  h1: Understanding the WizardLM-70B-V1.0 model
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 38]
```diff
@@ -0,0 +1,69 @@
+---
+meta:
+  title: WizardLM-70B-V1.0 model
+  description: This page explains the WizardLM-70B-V1.0 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:4]:
```suggestion
  description: This page provides information on the WizardLM-70B-V1.0 model
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 39]
```diff
@@ -0,0 +1,69 @@
...
+content:
+  h1: WizardLM-70B-V1.0 model
+  paragraph: This page explains the WizardLM-70B-V1.0 model
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:7]:
```suggestion
  paragraph: This page provides information on the WizardLM-70B-V1.0 model
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 40]
```diff
@@ -0,0 +1,69 @@
...
+```bash
+wizardlm-70B-V1.0
+```
+
+## Compatible Instances
+
+- TBD
+
+## Model introduction
+
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:32]: same as above

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 41]
```diff
@@ -0,0 +1,69 @@
...
+<Message type="tip">
+  Ensure you have the necessary libraries provided by NVIDIA for Triton to interact with the server from your client code.
+  You can find these libraries on [GitHub](https://github.com/triton-inference-server/client/tree/main).
+  Follow the instructions provided in the `README` file to integrate these libraries into your project.
+</Message>
+
+### Sending Inference requests
+
+To perform inference tasks with your LLM model deployed on the Triton Inference Server, follow these steps:
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:51]:
```suggestion
To perform inference tasks with your LLM model deployed on the Triton Inference Server, use the following command:
```

File: ai-data/llm-inference/concepts.mdx (hunk 2 above)
Comment (tgenaitay) [reply ai-data/llm-inference/concepts.mdx:12]: It's called Allowed IPs in our Console, so I think we should stay consistent with the concept na
Capture d’écran 2024-03-19 à 15 29 24
ming

**Review by pshambulingappa — APPROVED** [review]

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 42]
```diff
@@ -0,0 +1,87 @@
...
+```bash
+curl -s \
+-H "X-Auth-Token: <IAM token>" \
+-H "Content-Type: application/json" \
+--request POST \
+--url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/" \
+--data '{"text_input": "[INST]How can I use large language models to improve my customer service? [/INST]", "max_tokens": 200, "temperature": 0.2, "random_seed": 1, "top_p": 0.9}' | jq -r .text_output
+```
+
+Make sure to replace <IAM token> and <Deployment UUID> with your actual IAM token and the Deployment UUID you are targeting.
```
Comment (ldecarvalho-doc) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:61]:
```suggestion
Make sure to replace <IAM token> and <Deployment UUID> with your actual IAM token and the Deployment UUID you are targeting.
```
what is an IAM token? is it the API key? Best to replace it so we keep the names ISO all across the docs

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 43]
```diff
@@ -0,0 +1,87 @@
...
+[/INST]
+```
+
+### Receiving Inference responses
+
+Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed Inference server. 
+Process the output data according to your application's needs. The response will contain the output generated by the LLM model based on the input provided in the request.
+
+<Message type="note">
+  Despite efforts to ensure accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.
```
Comment (ldecarvalho-doc) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:86]: maybe add a link to the concept of "hallucinations" in this context, or explain it

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 44]
```diff
@@ -0,0 +1,77 @@
...
+```bash
+curl -s \
+-H "X-Auth-Token: <IAM token>" \
+-H "Content-Type: application/json" \
+--request POST \
+--url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/" \
+--data '{"text_input": "<s>[INST] Sing me a song about Scaleway [/INST] Model answer</s>", "max_tokens": 200, "temperature": 0.2, "random_seed": 1, "top_p": 0.9}' | jq -r .text_output 
+```
+
+Make sure to replace <IAM token> and <Deployment UUID> with your actual IAM token and the Deployment UUID you are targeting.
```
Comment (ldecarvalho-doc) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:58]: same here

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 45]
```diff
@@ -0,0 +1,81 @@
...
+```bash
+curl -s \
+-H "X-Auth-Token: <IAM token>" \
+-H "Content-Type: application/json" \
+--request POST \
+--url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/" \
+--data '{"text_input": "Scaleway Inference service. Say hello", "max_tokens": 200, "temperature": 0.2, "random_seed": 1, "top_p": 0.9}' | jq -r .text_output
+```
+
+Make sure to replace <IAM token> and <Deployment UUID> with your actual IAM token and the Deployment UUID you are targeting.
```
Comment (ldecarvalho-doc) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:58]: and here

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 46]
```diff
@@ -0,0 +1,81 @@
...
+ASSISTANT: I am WizardLM.</s>
+```
+
+### Receiving Inference responses
+
+Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed Inference server. 
+Process the output data according to your application's needs. The response will contain the output generated by the LLM model based on the input provided in the request.
+
+<Message type="note">
+  Despite efforts to ensure accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always verify the content generated independently .
```
Comment (ldecarvalho-doc) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:80]: same remark as before about hallucinations

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx (hunk 42 above)
Comment (tgenaitay) [reply ai-data/llm-inference/reference-content/llama-2-70b.mdx]: thank you @ldecarvalho-doc ! Replaced token by API key and linked to IAM API key how to.

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx (hunk 43 above)
Comment (tgenaitay) [reply ai-data/llm-inference/reference-content/llama-2-70b.mdx]: Great idea! Idea hallucinations in list of concepts, and linked to it when mentioned.

**Comment by tgenaitay** [comment]
I think this is as accurate and ready as it can be. Further proof-reading could help. 
I will let @seiteta give you the green light to merge and deploy next week, with hard deadline March 29 for go live.

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 47]
```diff
@@ -0,0 +1,32 @@
...
+  - ai-data
+---
+
+This documentation page shows you how to monitor your LLM Inference deployment with [Cockpit](/observability/cockpit/quickstart/).
+
+<Macro id="requirements" />
+
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+  - An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:22]: Cockpit activation is going to be deprecated on the 2nd of April. Cockpits will be automatically activated. Should we remove this requirement?

File: ai-data/llm-inference/quickstart.mdx [hunk 48]
```diff
@@ -0,0 +1,88 @@
+---
+meta:
+  title: LLM Endpoints - Quickstart
+  description: This page explains all the concepts related to Inference for LLMs
+content:
+  h1: LLM Endpoints - Quickstart
+  paragraph: This page explains all the concepts related to Inference for LLMs
+tags: 
+categories:
+  - ia
```
Comment (nerda-codes) [inline ai-data/llm-inference/quickstart.mdx:10]:
```suggestion
  - ai-data
```

**Comment by seiteta** [comment]
LGTM on my side.
Thanks @nerda-codes for the reviews.

**Pull request merged**